lib/reactive_table.html

```diff
@@ -94,12 +102,17 @@
                 <label class="previous-page">&lt;</label>&nbsp;&nbsp;
               {{/if}}
             {{/if}}
-            <label>{{i18n 'reactiveTable.page'}}&nbsp;<input class="form-control" type="text" value="{{getCurrentPage}}" />&nbsp;{{i18n 'reactiveTable.of'}} {{getPageCount}}</label>
+            <label>
+              <span>{{i18n 'reactiveTable.page'}}</span>
+              <input class="form-control" type="text" value="{{getCurrentPage}}" />
+              <span>{{i18n 'reactiveTable.of'}}</span>
+              <span class="page-number-count">{{getPageCount}}</span>
+            </label>
             {{#if isntLastPage}}
               {{#if useFontAwesome}}
-                &nbsp;&nbsp;<i class="next-page fa fa-caret-right"></i>
+                <i class="next-page fa fa-caret-right"></i>
               {{else}}
-                &nbsp;&nbsp;<label class="next-page">&gt;</label>
+                <label class="next-page">&gt;</label>
```

Comment:
Does removing the `&nbsp;`s change the spacing of this section, if you haven't added any additional css?